source/ore/gradle-deploy.rst

```diff
@@ -0,0 +1,86 @@
+==================
+Deploy from Gradle
+==================
+
+SpongeGradle_ makes it easy for plugin developers to deploy new versions of their plugin directly to Ore without even
+needing to open a browser. Once you have created a project on Ore and have published at least one version of your
+plugin, you can begin to deploy new versions of your plugin from your build script.
+
+.. tip::
+    For more info on how to get started with Gradle and Sponge, check out :doc:`/plugin/project/gradle`.
+
+Configuring the signing plugin
+==============================
+
+Like all Ore uploads, the artifacts you publish from Gradle must be signed with the PGP private key that corresponds to
+the public key you've added to your account in :doc:`/ore/publish`. Luckily, this is easily accomplished with Gradle as
+it comes with a core-plugin that does just this. In order to use the ``oreDeploy`` task in Gradle, *you must*
+configure the signing plugin as it depends on this specific plugin.
+
+For more information on how to configure the signing plugin, please refer to the Gradle User Guide entry:
+`The Signing Plugin <https://docs.gradle.org/current/userguide/signing_plugin.html>`__.
+
+Deployment credentials
+======================
+
+The first step in enabling direct deployment is generating a deployment key to let Gradle publish to your project
+remotely. You can do this by navigating to your project settings, scrolling down to "Deployment key" and clicking
+"Generate key".
+
+.. image:: /images/ore/help_3.png
+    :align: center
+    :alt: Deployment key
+
+.. warning::
+    Anyone with access to your special deployment key will have the ability to attempt to publish new versions to
+    your project. Of course, they will also need your private PGP key to do so but still...keep it secret, keep it safe!
+
+This deployment key must be supplied as the value of the property ``oreDeploy.apiKey``. Following best practice, you
+should put this in your ``gradle.properties`` file.
```

Comment:
You should document which `gradle.properties` file you are referring to. It can be used in the project root and in `~/.gradle/gradle.properties`, but it should be always configured in the user level one (otherwise you may accidentally commit it and make it available to the public.)